src/libraries/System.Net.Sockets/tests/FunctionalTests/Connect.cs

```diff
@@ -167,7 +164,7 @@ public async Task ConnectGetsCanceledByDispose()
                     disposedException = true;
                 }
 
-                if (usesApm)
+                if (UsesApm)
```

Comment:
The exception thrown by the `Task ConnectAsync` methods has changed. The exceptions are now consistent with other `Task`-returning `Socket` methods.